game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -351,7 +351,10 @@ private void gameDataChanged() {
           ServerConnectionProps.builder()
               .name(System.getProperty(TRIPLEA_NAME))
               .port(Integer.parseInt(System.getProperty(TRIPLEA_PORT)))
-              .password(System.getProperty(SERVER_PASSWORD))
+              .password(
+                  Optional.ofNullable(System.getProperty(SERVER_PASSWORD))
+                      .map(String::toCharArray)
+                      .orElse(null))
```

Comment:
Not sure if the benefit is that great if we don't ever clear the char array afterwards

Reply:
Agree. On one hand GC should kick in and at least destroy the array and at some point the memory would be overwritten.

For the most part it's a "use char[] where convenient", so don't go out of our way to use String. In this case overall I think/hope there is less conversion to string, ultimately simpler. To another extent, because we are reading password in from CLI args, coming in as a `String[]`, we are pretty boned in terms of the password being stored as a `String`